pkg/api/types.go

```diff
@@ -195,6 +195,119 @@ type VolumeSource struct {
 	NFS *NFSVolumeSource `json:"nfs"`
 }
 
+// Similar to VolumeSource but meant for the administrator who creates PVs.
+// Exactly one of its members must be set.
+type PersistentVolumeSource struct {
+	// GCEPersistentDisk represents a GCE Disk resource that is attached to a
+	// kubelet's host machine and then exposed to the pod.
+	GCEPersistentDisk *GCEPersistentDiskVolumeSource `json:"gcePersistentDisk"`
+	// HostPath represents a directory on the host.
+	// This is useful for development and testing only.
+	// on-host storage is not supported in any way
+	HostPath *HostPathVolumeSource `json:"hostPath"`
+}
+
+type PersistentVolume struct {
+	TypeMeta   `json:",inline"`
+	ObjectMeta `json:"metadata,omitempty"`
+
+	//Spec defines a persistent volume owned by the cluster
```

Comment:
OK, I understand.  Now is probably not the right time for this discussion, and I'm happy to defer it until aver v1.0, but I think that there's a lot of value in accommodating users sharing things with each other. And I think that whether a user wants to share something with someone else is an authorization concern, and orthogonal to what namespace the thing is in (along the lines of a file system, where things under my home directory ("namespace"), can be made readable to other users).  I suspect that the system is not set up this way today, in which case we should defer this discussion.